<html><head><title>Content</title></head><body>
<h1>Metabocloud home</h1>
<p>This is MetaboCloud a site that acts as an information medium on where to find metabolomics software and tools that were developed within the Netherlands Metabolomics Centre. Together the tools form a toolbox or <span style="line-height: 1.4em;">data support platform for processing and storage of data from metabolomics studies.</span></p>
<p><span style="line-height: 1.4em;">The vision of the NMC is to make software and tools available for the international (metabolomics) community. </span></p>
<p>The NMC toolbox contains two major components:</p>
<ol>
<li>a (metabolomics) study data capturing framework, and </li>
<li>data processing tools.</li>
</ol>
<p>The first contains a repository of metabolomics data, facilitating storage of raw and processed (a.o.) metabolomics data and study information, which can be searched and queried.</p>
<p>The data processing tools cover most of the metabolomics data processing workflow, including data preprocessing, quality control, statistical analysis, metabolite identification and biological interpretation tools.</p>
<h1>Data storage</h1>
<p>Source: <a href="https://github.com/NetherlandsMetabolomicsCentre/GSCF" target="_blank">github.com/NetherlandsMetabolomicsCentre/GSCF</a></p>
<p><span style="line-height: 1.4em;">The general study capturing framework (GSCF) is an application that can store any biological study. It contains templates which makes it possible to personalize and / or standardize the data upload and storage.</span></p>
<p>In order to allow flexibility to capture all information you require within a study, <em>and</em> to make it possible to compare studies or study data, the system uses customizable templates and ontologies. It is especially designed to store complex study designs including cross-over designs and challenges.</p>
<p>In addition to the storage of study design data, data of so-called assays can be added, organized in assay modules.</p>
<p>One implementation of the GSCF provides access for guests, providing a way to explore the functionalities of the framework on a few public studies: <a href="http://studies.dbnp.org/" target="_blank">studies.dbnp.org</a></p>
<p><span style="line-height: 1.4em;">The general study capturing framework facilitates sharing of data within a research group or consortium, as the study owner can decide who can view or access the data. In addition, the GSCF can stimulate collaborations by making study information publicly visible. New studies can be based on study data within the database, as standardized storage is stimulated by the system.</span></p>
<h1>Processing software</h1>
<p>As part of the <a href="http://metabolomicscentre.nl/" target="_blank">Netherlands Metabolomics Centre</a> a data processing infrastructure was developed, containing tools supporting the entire metabolomics workflow. Links to the tools that were developed in this context are gathered here, to provide one portal where these tools can be accessed. Most of the tools are also open source.</p>
<h2>Pre-Processing</h2>
<div id="node-256" class="node node-type-project build-mode-full clearfix">
<div id="projectcontent">
<div class="content">
<p>In the field of life sciences, LC-MS and GC-MS are among the most popular platforms to acquire data from human, microbial and plant tissues for food, nutrition and health applications. From these raw data, the metabolites can be derived that are most important and influential for a certain application. However, besides a proper data quality and analysis strategy, this also requires a preprocessing step that allows extracting information of pure components from the data. </p>
</div>
</div>
</div>
<h3>mzExtract</h3>
<p>The challenge with untargeted, global profiling metabolomics acquisition methods is the proper and automated extraction and integration of relevant metabolite features. mzExtract is a new algorithm that enables untargeted integration of samples that are measured with high resolution LC-MS. </p>
<p>SOURCE: <a href="https://github.com/NetherlandsMetabolomicsCentre/mzExtractGui" target="_blank">github.com/NetherlandsMetabolomicsCentre/mzExtract</a></p>
<p>REFERENCE:</p>
<p><a href="http://dx.doi.org/10.1016/j.aca.2013.09.028" target="_blank">A new approach to untargeted integration of high resolution liquid chromatography-mass spectrometry data., van der Kloet F.M., Hendriks M., Hankemeier T., Reijmers T., Analytica Chimica Acta, Volume 801, (2013), pages 34-42.</a> </p>
<h3>mzQual</h3>
<p>Hyphenated separation mass spectroscopy techniques are major platforms in metabolomics. A wide range of compounds can be measured (semi) quantitatively over a huge concentration range in a highly sensitive and selective manner. To obtain data of high quality (accurate, reproducible, high signal-to-noise), especially when a large number of samples have to be analysed, needs an elaborate quality control strategy.</p>
<p><span style="line-height: 1.4em;">The mzQual tool provides a fully automated control strategy, that can distinguish the various sources of variation. mzQual flags, and provides means for visual inspection of, suspicious samples or metabolites. Furthermore, the tool provides automated correction for different</span><span style="line-height: 1.4em;"> types of (unwanted) analytical variation, such as blank correction, internal standard correction and between- and within-batch correction. Finally, it provides figures of merit that quantify the original and final quality of the data.</span></p>
<p>SOURCE: <a href="https://github.com/NetherlandsMetabolomicsCentre" target="_blank">github.com/NetherlandsMetabolomicsCentre/mzQual</a></p>
<p>REFERENCE:</p>
<h3>LCMS TNO-DECO</h3>
<p> <span style="line-height: 1.4em;">In the field of life sciences, LC-MS and GC-MS are among the most popular platforms to acquire data from human, microbial and plant tissues for food, nutrition and health applications. From these raw data, the metabolites can be derived that are most important and influential for a certain application. However, besides a proper data quality and analysis strategy, this also requires a preprocessing step that allows extracting information of pure components from the data. Deconvolution is an approach to perform such tasks.</span></p>
<p>The starting point of this project is an existing tool for the deconvolution of GC-MS data (i.e. TNO-DECO, see Jellema et al, 2010. Chemometrics and intelligent laboratory systems 104: 132). Its unique building blocks are used to build a generic and automatic pre-processing tool for high resolution (HR) LC-MS and GC-MS data.</p>
<p>SOURCE: </p>
<p>REFERENCES:</p>
<p><a href="http://onlinelibrary.wiley.com/doi/10.1002/rcm.6517/abstract" target="_blank">Pre-processing liquid chromatography/high-resolution mass spectrometry data: extracting pure mass spectra by deconvolution from the invariance of isotopic distribution. <span class="highlight" style="line-height: 1.4em;">Krishnan</span><span style="line-height: 1.4em;"> S.</span><span style="line-height: 1.4em;">, Verheij E.E.<span style="line-height: 1.4em;">, </span>Bas R.C.<span style="line-height: 1.4em;">, </span></span><span class="highlight" style="line-height: 1.4em;">Hendriks</span><span style="line-height: 1.4em;"> M.M.W.B.<span style="line-height: 1.4em;">, </span>Hankemeier T.<span style="line-height: 1.4em;">, </span>Thissen U.<span style="line-height: 1.4em;">, </span>Coulier L<span style="line-height: 1.4em;">., <span>Rapid Commun Mass Spectrom.</span><span> 27 (2013) 917-923</span></span></span></a></p>
<p><a style="line-height: 1.4em;" href="http://www.sciencedirect.com/science/article/pii/S0003267012008811" target="_blank">Instrument and process independent binning and baseline correction methods for liquid chromatography-high resolution-mass spectrometry deconvolution. Krishnan<span style="line-height: 1.4em;"> S., Vogels J.T., Coulier L., Bas R.C., </span>Hendriks<span style="line-height: 1.4em;"> M.M.W.B., Hankemeier T., Thissen U. </span><span class="jrnl" style="line-height: 1.4em;" title="Analytica chimica acta">Anal Chim Acta</span><span style="line-height: 1.4em;">. 740 (2012) 12-19</span></a></p>
<p><span style="line-height: 1.4em;"><span style="line-height: 1.4em;"><span><br></span></span></span></p>
<h2>Biostatistics</h2>
<p>Within the Biostatistics theme of the NMC new statistical methodology is developed that will serve in extracting information from metabolomics data. The product of some of the projects were implemented as Matlab or R algorithms. Most algorithms relate to the <span style="line-height: 1.4em;">analysis of metabolomics data with a complex structure and network analysis. </span></p>
<h3>Simplivariate methods</h3>
<p><span style="line-height: 1.4em;">Simplivariate methods can be used for an exploratory analysis of high-dimensional functional genomics data. It can be used in addition to Cluster Analysis and Principal Component Analysis. The advantage of simplivariate methods is their ability to generate simple and interpretable solutions. It is based on the observation that functional genomics data often contain both informative and non-informative variation, we propose a method that finds sets of variables containing informative variation. This informative variation is subsequently expressed in easily interpretable simplivariate components.</span></p>
<p><span style="line-height: 1.4em;">The informative variation is described by multiplicative models of the informative variables that can adequately represent the relations between functional genomics data.</span></p>
<p><span style="line-height: 1.4em;">The </span><a style="line-height: 1.4em;" href="http://www.bdagroup.nl/content/Downloads/software/software.php">software</a><span style="line-height: 1.4em;"> and additional information can be downloaded from the website of the </span><a style="line-height: 1.4em;" href="http://www.bdagroup.nl">Biosystems Data Analysis</a><span style="line-height: 1.4em;"> group of the University of Amsterdam. </span><span style="line-height: 1.4em;">Additionally, the Matlab</span><sup style="line-height: 1.4em;">®</sup><span style="line-height: 1.4em;"> Genetic Algorithm and Direct search Toolboxes are required.</span></p>
<p> </p>
<p>REFERENCES<br><a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0020747">Simplivariate Models: Uncovering the Underlying Biology in Functional Genomics Data Saccenti E, Westerhuis JA, Smilde AK, van der Werf MJ, Hageman JA, Hendriks MMWB PloS ONE, Volume 6(6) (2011), article number e20747</a></p>
<p><br><a href="http://onlinelibrary.wiley.com/doi/10.1002/cem.1411/abstract;jsessionid=E76176E50894F0E61A5342A32FCE62C4.d04t03">Tracy-Widom statistic for the largest eigenvalue of autoscaled real matrices Edoardo Saccenti, Age K. Smilde, Johan A. Westerhuis, Margriet M. W. B. Hendriks Journal of Chemometrics, Volume 25(12) (2011), pages 644 - 65</a></p>
<h3>MetStaT R-package</h3>
<p>Source: <a href="http://cran.r-project.org/web/packages/MetStaT/index.html" target="_blank">cran.r-project.org/web/packages/MetStaT</a></p>
<p>This R-package contains a number of metabolomics statistical tools developed within the Netherlands Metabolomics Centre and can be found on the cran R repository. The package contains code for</p>
<ol>
<li>ANOVA Simultaneous Component Analysis, </li>
<li>Figures of Merit,</li>
<li>Calculating the Q statistic of Goeman’s global test for metabolomic pathways,</li>
<li>Reverse engineering of networks</li>
</ol>
<p> </p>
<p>REFERENCES ASCA</p>
<p><a href="http://bioinformatics.oxfordjournals.org/content/21/13/3043.full" target="_blank">Smilde AK, Jansen JJ, Hoefsloot HCJ, Lamers R JAN, van der Greef J, Timmerman ME. ANOVA simultaneous component analysis (ASCA): a new tool for analyzing designed metabolomics data. Bioinformatics 21, (2005), p. 3043 - 3048</a></p>
<p><a href="http://onlinelibrary.wiley.com/doi/10.1002/cem.1400/full" target="_blank">Zwanenburg G, Hoefsloot HCJ, Westerhuis JA, Jansen JJ and Smilde AK, ANOVA principal component analysis and ANOVA simultaneous component analysis: a comparison. J Chemometrics, 25, (2011), p. 561 - 567</a></p>
<p> </p>
<p>REFERENCE Figures of Merit</p>
<p><a href="http://pubs.acs.org/doi/abs/10.1021/ac102374c" target="_blank">New Figures of Merit for Comprehensive Functional Genomics Data: The Metabolomics Case. Van Batenburg MF, Coulier L, van Eeuwijk F, Smilde AK, Westerhuis JA. Analytical Chemistry, Volume 83(9) (2011), pages 3267-3274</a></p>
<p> </p>
<p>REFERENCE Q statistic</p>
<p><a href="Global%20test%20for%20metabolic%20pathway%20differences%20between%20conditions.%20Diana%20M.%20Hendrickx,%20Huub%20C.J.%20Hoefsloot,%20Margriet%20M.W.B.%20Hendriks,%20Andre.%20Canelas%20and%20Age%20K.%20Smilde.%20Analytica%20Chimica%20Acta,%20Volume%20719%20(2012),%20pages%208%20-%2015" target="_blank">Global test for metabolic pathway differences between conditions. Hendrickx DM, Hoefsloot HCJ, Hendriks MMWB, Canelas A and Smilde AK. Analytica Chimica Acta, Volume 719 (2012), pages 8 - 15</a></p>
<p> </p>
<p>REFERENCE Reverse engineering of networks</p>
<p><a href="http://pubs.rsc.org/en/content/articlehtml/2011/mb/c0mb00083c" target="_blank">Reverse engineering of metabolic networks, a critical assessment. Hendrickx DM, Hendriks MMWB, Eilers PHC, Smilde AK and Hoefsloot HCJ. Mol. BioSyst, Volume 7:2 (2011) pages 511-520</a></p>
<h2>Identification</h2>
<div class="content">
<div>One major bottleneck in metabolomics is identification of all the metabolites observed in the biological samples measured. As a result the development of innovative identification methods is mandatory. Within the NMC multiple metabolite identification projects had been defined aimed at improving the current state of metabolite identification strategies. Main priorities of these research projects were the construction of MS<sup>n</sup> and NMR databases of reference metabolites, the development of new analytical methods (improved MS<sup>n</sup> acquisition, LC-MS<sup>n</sup> acquisition, automatic fractionation and enrichment of compounds present at low levels to obtain sufficient quantities for MS<sup>n</sup> and NMR measurements) and the development of innovative computational tools (MS<sup>n</sup> preprocessing and processing as well as NMR spectrum prediction, spectrum retrieval, visualization and NMR-based compound quantification tools). </div>
<div> </div>
<div>Important results were obtained with respect to databases: an advanced NMR/MS database has been constructed containing circa 6000 polyphenol/flavonoid compounds. In collaboration with the Humane Metabolome Database (David Wishart) a MS<sup>n</sup> spectral data base has been constructed consisting of circa 430 reference compounds. In addition, essential tools for MS<sup>n</sup> data preprocessing and processing have been created. Tool development has also been carried out for spectrum visualization of reference spectra present in the MS<sup>n</sup> and NMR databases as well as for structure annotation to unknown metabolites. One of the strategic projects in metabolite identification is computer-assisted identification by integration of a wide range of spectroscopic, physicochemical and biochemical data sources. Spectroscopic data are used for comprehensive structure generation followed by fast candidate rejection, aimed at leaving only a few candidate metabolites. </div>
</div>
<h3>MetiTree</h3>
<p>URL: <a href="http://www.metitree.nl" target="_blank">www.metitree.nl</a></p>
<p>Source: <a href="https://github.com/NetherlandsMetabolomicsCentre/metitree" target="_blank">github.com/NetherlandsMetabolomicsCentre/metitree</a></p>
<p id="article-title-1">MetiTree: a web application to organize and process high resolution multi-stage mass spectrometry metabolomics data.</p>
<p>Identification of metabolites using high resolution multistage mass spectrometry (MS<sup>n</sup>) data is a significant challenge demanding access to all sorts of computational infrastructures. MetiTree is a user-friendly, web application dedicated to organize, process, share, visualize, and compare MS<sup>n</sup> data. It integrates several features to export and visualize complex MS<sup>n</sup> data, facilitating the exploration and interpretation of metabolomics experiments. A dedicated spectral tree viewer allows the simultaneous presentation of three related types of MS<sup>n</sup> data, namely, the spectral data, the fragmentation tree, and the fragmentation reactions. MetiTree stores the data in an internal database to enable searching for similar fragmentation trees and matching against other MS<sup>n</sup> data. As such MetiTree contains much functionality that will make the difficult task of identifying unknown metabolites much easier.</p>
<p> </p>
<p>REFERENCE:</p>
<p><a href="http://bioinformatics.oxfordjournals.org/content/28/20/2707" target="_blank">MetiTree: a web application to organize and process high-resolution multi-stage mass spectrometry metabolomics data. Rojas-Chertó M., van Vliet M., Peironcely J.E., van Doorn R., Kooyman M., te Beek T., van Driel M.A., Hankemeier T., Reijmers T., Bioinformatics, Volume 28(20), (2012), pages 2707-2709.</a> </p>
<h3>MetIDB</h3>
<p><span style="line-height: 1.4em;"><br></span></p>
<p><span style="line-height: 1.4em;">URL: </span><a style="line-height: 1.4em;" href="http://www.metidb.org" target="_blank">www.metidb.org</a></p>
<p><span style="line-height: 1.4em;">MetIDB is a reference database of experimental and predicted </span><span style="line-height: 1.4em;">1</span><span style="line-height: 1.4em;">H NMR spectra of 6000 flavonoids. By incorporating the stereochemistry, intramolecular interactions, and solvent effects into the prediction model, chemical shifts and couplings were predicted with great accuracy. The reference database has a user-friendly web-based interface providing various interfaces to the data and data-mining possibilities. For each compound, additional information is available comprising compound annotation, a </span><span style="line-height: 1.4em;">1</span><span style="line-height: 1.4em;">H NMR spectrum, 2D and 3D structure with correct stereochemistry, and monoisotopic mass as well as links to other web resources. </span><span style="line-height: 1.4em;">Using this database, the process of flavonoid identification can then be significantly shortened by avoiding repetitive elucidation of already described compounds.</span></p>
<p><span style="line-height: 1.4em;"> </span></p>
<p><a href="http://pubs.acs.org/doi/abs/10.1021/ac4016837" target="_blank">MetIDB: A Publicly Accessible Database of Predicted and Experimental 1H NMR Spectra of Flavonoids, <span style="line-height: 1.4em;">Mihaleva VV, te Beek TAH, van Zimmeren F, Moco S, Laatikainen R, Niemitz M, Korhonen S-P, van Driel MA and J Vervoort, </span>85<span style="line-height: 1.4em;">(18), 2013, 8700-8707</span></a></p>
<p><span style="line-height: 1.4em;"><br></span></p>
<h3>MEF</h3>
<p><span style="font-size: 16px;"><span style="font-family: Calibri;"><span style="font-size: 18px;">The Multistage Elemental Formula (MEF) tool allows assignment of correct elemental compositions to compounds, their fragment ions and neutral losses that originate from the molecular ion by using multistage mass spectrometry (MS<sup>n</sup>). MEF reduces the list of predicted elemental compositions for each ion by analyzing the elemental compositions of its parent (precursor ion) and descendants</span> <span style="font-size: 18px;">(fragments). </span></span></span><span style="font-size: 18px;"><span style="font-family: Calibri;"><br></span></span><span style="font-size: 16px;"><span style="font-family: Calibri;"><br></span></span></p>
<p><span style="font-size: 16px;"><span style="font-family: Calibri;">SOURCE: </span></span></p>
<ul>
<li><span style="font-size: 16px;"><span style="font-family: Calibri;"><a href="http://dl.dropbox.com/u/3893174/MEF.zip" target="_blank">MEF tool</a> </span></span></li>
<li><span style="font-size: 16px;"><span style="font-family: Calibri;"><a href="http://dl.dropbox.com/u/3893174/MEF-source.zip" target="_blank">MEF source</a></span></span></li>
</ul>
<p> <span style="font-size: 16px;"><span style="font-family: Calibri;"><br></span></span></p>
<p><span style="font-size: 16px;"><span style="font-family: Calibri;">REFERENCE:<br></span></span></p>
<p><span style="font-size: 16px;"><span style="font-family: Calibri;"><a href="http://bioinformatics.oxfordjournals.org/content/27/17/2376" target="_blank">Elemental composition determination based on MS<sup>n</sup>. Rojas-Cherto M., Kasper P.T., Willighagen E.L., Vreeken R.J., Hankemeier T., Reijmers T.H., Bioinformatics, Volume 27(17) (2011), pages 2376-2383.</a> </span></span></p>
<p><span style="font-size: 16px;"><span style="font-family: Calibri;"><br></span></span></p>
<h3>SAMSN</h3>
<p>Multistage mass spectrometry (MS<sup>n</sup>) generating so called spectral trees is a powerful tool in the annotation and structural identification of metabolites and is increasingly used in the area of accurate mass LC/MS-based metabolomics to identify unknown compounds. As a consequence, there is a growing need for computational tools specifically designed for the processing and interpretation of MS<sup>n</sup> data.</p>
<p>The SAMSN (Structure Assignment MS<sup>n</sup>) tool is an extended version of the MEF tool. Besides processing of MS<sup>n</sup> spectral trees from fragmentation spectra into fragmentation trees (assignment of elemental compositions to all ions and neutral losses) it allows the extraction of MS<sup>n</sup> fragmentation tree fingerprints and the calculation of MS<sup>n</sup> fragmentation tree similarity values.  </p>
<p> </p>
<p>SOURCE: <a href="https://github.com/NetherlandsMetabolomicsCentre/samsn" target="_blank">github.com/NetherlandsMetabolomicsCentre/samsn</a></p>
<p> </p>
<p>REFERENCE:</p>
<p><a href="http://pubs.acs.org/doi/full/10.1021/ac2034216" target="_blank">Metabolite identification using automated comparison of high-resolution multistage mass spectral trees. Rojas-Chertó M., Peironcely J.E., Kasper P.T., van der Hooft J.J.J., de Vos R.C.H., Vreeken R., Hankemeier T., Reijmers T., Analytical Chemistry, Volume 84(13), (2012), pages 5524-5534.</a></p>
<h3>OMG</h3>
<p>OMG (Open Molecule Generator) is the first open source structure generator which for a given elemental composition produces all non-isomorphic chemical structures that match that elemental composition. As additional input OMG can accept one or multiple (non-overlapping) prescribed substructures. As such OMG is especially of great importance for metabolomics, where identifying unknown metabolites is still a major bottleneck.  </p>
<p>SOURCE: <a href="http://sourceforge.net/projects/openmg/" target="_blank">sourceforge.net/projects/openmg/</a></p>
<p>REFERENCE:</p>
<p><a href="http://www.jcheminf.com/content/4/1/21" target="_blank">OMG: Open Molecule Generator, Peironcely J.E., Rojas-Chertó M., Fichera D., Reijmers T., Coulier L., Faulon J.-L., Hankemeier T., Journal of Cheminformatics, Volume 4(September),  (2012), 21.</a></p>
<h3>PMG</h3>
<p>PMG (Parallel Molecule Generator) is a Java-based implementation of an open source multi-core automatic molecular structure generator, especially designed for metabolomics. It is an extended version of OMG, the earlier released structure generator. Due to improved speed it accommodates prescribed substructures and it removes unstable structures using a bad list of rings and substructures that are not found in human metabolites.</p>
<p> </p>
<p>SOURCE: <a href="http://sourceforge.net/projects/pmgcoordination/" target="_blank">sourceforge.net/projects/pmg</a>.</p>
<p> </p>
<p>REFERENCE:</p>
<p><a href="http://dx.doi.org/10.1016/j.entcs.2013.11.005" target="_blank">PMG: Multi-core metabolite identification., Jaghoori M.M., Jongmans S.T.Q., de Boer F., Peironcely J., Faulon J.-L., Reijmers T., Hankemeier T., Electronic Notes in Theoretical Computer Science, 299 (2013) 53-60.</a></p>
<h3>Metabolite Likeness</h3>
<p>In order to understand the chemical space spanned by endogenous metabolites in more detail human metabolites and non-metabolites were analyzed for chemical features that are characteristic for both classes of compounds. In addition it was checked which molecular descriptors and classifiers were capable of distinguishing metabolites from non-metabolites by assigning a 'metabolite-likeness' score.</p>
<p>SOURCE: <a href="http://sourceforge.net/projects/metabolikeness/" target="_blank">sourceforge.net/projects/metabolikeness</a></p>
<p>REFERENCE:</p>
<p><a href="http://dx.doi.org/10.1371/journal.pone.0028966" target="_blank">Understanding and classifying metabolite space and metabolite-likeness., Peironcely J.E., Reijmers T., Coulier L., Bender A., Hankemeier T., PLoS ONE, Volume 6(12), (2011), e28966. </a></p>
<h1>Links to others</h1>
<p>Related software for Metabolomics data processing and storage can also be found elsewhere. Below you will find a couple of links to interesting sites / repositories of metabolomics tools and software.</p>
<p> </p>
<p>Storage</p>
<ul>
<li><a href="http://www.ebi.ac.uk/metabolights/">MetaboLights</a></li>
</ul>
<p><span style="line-height: 1.4em;">Pre-processing</span></p>
<ul>
<li><a href="http://mzmine.sourceforge.net/">MZmine 2</a></li>
<li><a href="http://metlin.scripps.edu/xcms/">XCMS</a></li>
<li><a href="http://www.wageningenur.nl/nl/show/MetAlign.htm">MetAlign</a></li>
</ul>
<p>Biostatistics</p>
<ul>
<li><a href="http://www.bdagroup.nl/content/Downloads/software/software.php">BDA group, University of Amsterdam</a></li>
<li><a href="http://www.models.kvl.dk/algorithms">Q&amp;T group, University of Copenhagen</a></li>
</ul>
<p>Identification</p>
<h1>News01</h1>
<h4>NMC</h4>
<p>The Netherlands Metabolomics Centre focusses on the development of metabolomics based technologies and instrumentation to address the current and future challenges in biology, biotechnology and biomedical research in order to improve personalised health and quality of life.</p>
<p>visit <a href="http://www.metabolomicscentre.nl/" target="_blank">www.metabolomicscentre.nl</a> for more information</p>
<h1>News02</h1>
<h4>What is Metabolomics</h4>
<p>Much like genomics aims to unravel the structure of the genome, metabolomics focuses on understanding the many small molecule metabolites that result from a cell’s metabolic processes.</p>
<p><span style="line-height: 1.4em;">For research, the study of metabolomics provides the means to measure the effects of a variety of stimuli on individual cells, tissues, and bodily fluids.</span></p>
<p>By studying how their metabolic profiles change with the introduction of chemicals or the expression of known genes, for example, researchers can more effectively study the immediate impact of disease, nutrition, pharmaceutical treatment, and genetic modifications while using a systems biology approach.</p>
<h1>News03</h1>
<h4><span style="line-height: 1.4em;">Software/Tools</span></h4>
<p>The software and tools are mostly distributed to other sites. Therefore, for all tools you will find links to the places were the software can be found. For distribution we use the website of the contributing groups and Github.</p>
<h4>Github</h4>
<p>Most of our software projects are available on Github. This allows you to download, use or even contribute to our codebase. Also improvements or new features can be requested by using the project issue tracker.</p>
<p><a href="https://github.com/NetherlandsMetabolomicsCentre" target="_blank">github.com/NetherlandsMetabolomicsCentre</a></p>
</body></html>